pages/ksphere/kommander/1.1.0-beta/operations/identity-providers/index.md

```diff
@@ -22,34 +28,44 @@ You can configure as many Identity Providers as you like and users will be able
 
 ![Identity](/ksphere/kommander/1.1.0-beta/img/Identity-providers-table.png)
 
-Figure 1 - Identity Providers
+Identity Providers
 
 #### Limiting who has access:
 
 - The Github provider allows to specify which orgs and teams are eligible for access.
 
 ![Github Form](/ksphere/kommander/1.1.0-beta/img/Identity-provider-Github.png)
 
-Figure 2 - Github Form
+Github Form
 
 - The LDAP provider allows to configure search filters for either users or groups.
 
 ![LDAP Form](/ksphere/kommander/1.1.0-beta/img/Identity-provider-LDAP.png)
 
-Figure 3 - LDAP Form
+LDAP Form
 
 - The OIDC provider cannot limit users based on identity.
 
 ![OIDC Form](/ksphere/kommander/1.1.0-beta/img/Identity-provider-OIDC.png)
 
-Figure 4 - OIDC Form
+OIDC Form
 
-#### Temporarily disabling a provider
+### Temporarily disabling a provider
 
-Untick the checkbox labelled “enabled” on the Identity Providers table. The provider option will no longer appear on the login screen.
+Open the actions menu on the Identity Providers table and click Disable. The provider option will no longer appear on the login screen.
 
-- The OIDC provider cannot limit users based on identity.
+![Identity Provider Table Row Action Menu](/ksphere/kommander/1.1.0-beta/img/Identity-provider-table-action-menu.png)
+
+Identity Provider Table Row Action Menu
+
+## Groups
+
+Access control groups are configured in the Groups tab of the Identity Providers page. Refer to [Access Control](https://docs.d2iq.com/ksphere/kommander/1.1.0-beta/operations/access-control/) for an overview of groups in Kommander.
+
+![Identity Provider Groups](/ksphere/kommander/1.1.0-beta/img/access-control-idp-groups.png)
 
-![Enabled Checkbox](/ksphere/kommander/1.1.0-beta/img/Identity-provider-enabled-checkbox.png)
+Identity Provider Groups
 
-Figure 5 - Enabled Checkbox
+## Related Information
+- [Authorize a group across clusters](https://docs.d2iq.com/ksphere/kommander/1.1.0-beta/tutorials/authorize-all-users/)
+- [Granting access to Kubernetes resources in Kommander](https://docs.d2iq.com/ksphere/kommander/1.1.0-beta/tutorials/configure-rbac/)
```

Comment:
The previous link works for me, but this one doesn't. Is that expected?

Reply:
It should work once #2985 merges